src/main/resources/edu/cornell/kfs/sys/businessobject/datadictionary/UserProcurementProfile.xml

```diff
@@ -149,19 +162,31 @@
     
   </bean>
   
-        <bean id="UserProcurementProfile-profileUser.principalName" parent="UserProcurementProfile-profileUser.principalName-parentBean"/>
-
-  <dd:boAttributeRef id="UserProcurementProfile-profileUser.principalName-parentBean" parent="PersonImpl-principalName" abstract="true" 
-  attributeName="profileUser.principalName" label="User Principal Name" shortLabel="Principal Name" required="false">
-    <dd:user universalIdAttribute="principalId" userObjectAttribute="profileUser"/>
-  </dd:boAttributeRef>
-    
-    <bean id="UserProcurementProfile-profileUser.name" parent="UserProcurementProfile-profileUser.name-parentBean"/>
-
-  <dd:boAttributeRef id="UserProcurementProfile-profileUser.name-parentBean" parent="PersonImpl-name" abstract="true" 
-  attributeName="profileUser.name" label="Person Name" shortLabel="Person Name" required="false">
-    <dd:user universalIdAttribute="principalId" userObjectAttribute="profileUser"/>
-  </dd:boAttributeRef>
+  <bean id="UserProcurementProfile-profileUser.principalName"
+        parent="UserProcurementProfile-profileUser.principalName-parentBean"/>
+  <bean id="UserProcurementProfile-profileUser.principalName-parentBean" abstract="true"
+        parent="PersonImpl-principalName"
+        p:name="profileUser.principalName" p:label="User Principal Name"
+        p:shortLabel="Principal Name" p:required="false">
+      <property name="control">
+          <bean parent="KualiUserControlDefinition" p:personNameAttributeName="profileUser.principalName"
+                p:universalIdAttributeName="principalId"
+                p:userIdAttributeName="profileUser.principalName"/>
+      </property>
+  </bean>
+  
+  <bean id="UserProcurementProfile-profileUser.name"
+        parent="UserProcurementProfile-profileUser.name-parentBean"/>
+  <bean id="UserProcurementProfile-profileUser.name-parentBean" abstract="true"
+        parent="PersonImpl-name"
+        p:name="profileUser.name" p:label="Person Name"
+        p:shortLabel="Person Name" p:required="false">
+      <property name="control">
+          <bean parent="KualiUserControlDefinition" p:personNameAttributeName="profileUser.name"
+                p:universalIdAttributeName="principalId"
+                p:userIdAttributeName="profileUser.name"/>

```

Comment:
Should the userIdAttributeName be referencing "profileUser.principalName" instead?